### PR TITLE
New method names.  sample_linf_ball restricted to lam_domain.

### PR DESCRIPTION
**Pull request by scottw13**
Changed some method names, cleaned up some for loops, changed some variable names.  Restrict sample_linf_ball to only samples inside lam_domain.

### CODE REVIEW

**Comment by lcgraham** [comment]
This should be added to the dev (and/or dev_sensitivity) branch. Modules and subpackages must have tests written for them before being added to the master branch.

File: bet/sensitivity/__init__.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
```
Comment (lcgraham) [inline bet/sensitivity/__init__.py]: Add a short description of the subpackage here.

File: bet/sensitivity/gradientsRBF.py [hunk 2]
```diff
@@ -0,0 +1,412 @@
+
```
Comment (lcgraham) [inline bet/sensitivity/gradientsRBF.py]: Add copyright line to top of all files.

**Comment by scottw13** [comment]
Done.  Should I make a new pull request to dev-sensitivity rather than master?

**Comment by lcgraham** [comment]
Thanks, Yes to either dev-sensitivity or dev unless you can switch the
branch to be merged into for an open pull request.

**Pull request closed without merging**